content/javascript/concepts/objects/terms/getOwnPropertySymbols/getOwnPropertySymbols.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: 'Object.getOwnPropertySymbols()'
+Description: 'Returns an array of all symbol property found directly upon a given object.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Properties'
+  - 'Objects'
+  - 'Functions'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/create-a-back-end-app-with-javascript'
+---
+
+The **`Object.getOwnPropertySymbols()`** method is a static method that returns an array of all symbol property found directly upon a given object. If the object has no own symbol properties initially, **`Object.getOwnPropertySymbols()`** returns an empty array.
+
+> **Note:** **`Object.getOwnPropertySymbols()`** method itself does not contain the symbol properties of an object and only the string properties.
+
+## Syntax
+
+```pseudo
+Object.getOwnPropertySymbols(object_name)
+```
+
+- `object_name`: The name of an object whose symbol properties are to be returned.
+
+## Example 1
+
+In the following code, `Object.getOwnPropertySymbols(object_name)` will return an array of all symbol properties that have been found:
+
+```js
+var Electrical_Devices = {
+  group1: ['Fridge', 'Microwave', 'Rice Cooker', 'Washing Machine'],
+  //add symbol
+  [Symbol('😊')]: 'localSymbol',
+  [Symbol('4554tu')]: 'globalSymbol',
+};
+console.log(Object.getOwnPropertySymbols(Electrical_Devices));
+```
+
+This will return the following output:
+
+```shell
+[ Symbol(😊), Symbol(4554tu) ]
+```
+
+## Example 2
+
```

Comment:
give a small description here of what the reader can expect from the coming code snippet like you did above